python/tskit/trees.py

```diff
@@ -9349,11 +9349,17 @@ def segregating_sites(
         What is computed depends on ``mode``. For a sample set ``A``, computes:
 
         "site"
-            The sum over sites of the number of alleles found in ``A`` at each site
-            minus one, per unit of chromosome length.
-            If all sites have at most two alleles, this is the density of sites
-            that are polymorphic in ``A``. To get the **number** of segregating minor
-            alleles per window, pass ``span_normalise=False``.
+            The sum over sites of [the number of alleles found in ``A`` at each site
+            minus one], per unit of chromosome length.
+            If all sites have at most two alleles in ``A``,
+            this is the density of segregating/polymorphic sites in ``A``
+            (since the "minus one" reduces the sum for ancestral allele).
+            Hence, multiple mutations per site increase the
+            returned value, but only if they segregate in ``A``.
+            If an ancestral allele does not segregate in ``A``,
+            the "minus one" will reduce the sum for another allele.
```

Comment:
```suggestion
            If all sites have at most two alleles in ``A``,
            this is the density of segregating/polymorphic sites in ``A``
            (since the "minus one" reduces the sum for monoallelic sites).
            For sites with more than two alleles, the sum is increased by
            one for each additional allele segregating in ``A``.
```
How about this?

Reply:
I like it!